docs/ORGANIZATION.md

```diff
@@ -1,21 +1,108 @@
+API docs, guides, plugins, roadmaps, releases, developer guides, developer
+resources, how-to guides and community resources live in the [`napari/napari`]
+repo under the `docs` directory. For backwards compatibility with the previous
+structure as used in [`napari/docs`], some files are "duplicated" by using the
+[MyST `include` directive](https://myst-parser.readthedocs.io/en/latest/sphinx/use.html#include-a-file-from-outside-the-docs-folder-like-readme-md).
 
-Tutorials, the main index page, and the WIP sphinx theme are in the [`napari/napari.github.io`] repo.
+Tutorials, the main index page, and the napari sphinx theme are in the
+[`napari/napari.github.io`] repo.
 
 The [`napari/docs`] repo contains no original documentation sources.
 
 ## Bringing it all together
 
-For the [`napari/docs`] repo, files are built in `docs/_build/html` in the main [`napari/napari`] repo and then copied over to the appropriate version number or `dev` using an automated process.
+For the [`napari/docs`] repo, files are built in `docs/_build/html` in the main
+[`napari/napari`] repo and then copied over to the appropriate version number or
+`dev` using an automated process.
 
-Documentation sources are copied over to `napari.github.io` using the `copy-docs.py` script (found in the `docs` directory of [`napari/napari`]). Duplicate files are excluded and the table of contents is automatically updated. Specifying which files should be copied / excluded can be modified in the script itself. This script is automatically run in continuous integration and the changes are pushed to the [`napari/napari.github.io`] repo, which will then automatically build and update the site.
+Documentation sources are copied over to `napari.github.io` using the
+`copy-docs.py` script (found in the `docs` directory of [`napari/napari`]).
+Duplicate files are excluded and the table of contents is automatically updated.
+Specifying which files should be copied / excluded can be modified in the script
+itself. This script is automatically run in continuous integration and the
+changes are pushed to the [`napari/napari.github.io`] repo, which will then
+automatically build and update the site.
+
+## Making changes to the documentation
+
+For any changes in documentation (except the tutorials), a PR should be
+submitted to the [`napari/napari`] repo. When a new file is added to this repo
+under the current folder structure, it will automatically be copied over to
+[`napari/napari.github.io`] and the [napari.org] table of contents will be
+automatically updated. If the changes are done directly on
+[`napari/napari.github.io`], they will be overwritten when a new PR is merged to
+[`napari/napari`].
+
+If any autogenerated documentation sources are added to [`napari/napari`], they
```

Comment:
Might be good to give an example here like "See the preferences documentation for an example"

Reply:
I've added this, but currently the preferences docs are not rendered in napari.org (see #3812)